src/tools/info/ucx_info.c

```diff
@@ -183,7 +184,7 @@ int main(int argc, char **argv)
     }
 
     if (print_opts & (PRINT_UCP_CONTEXT|PRINT_UCP_WORKER|PRINT_UCP_EP)) {
-        if (ucp_features == 0) {
+        if (ucp_features == 0 && ucp_ep_params.field_mask == 0) {
```

Comment:
1. code style requires ( ) around each boolean sub-expression in && expression
2. need to update help and error message to indicate which flags in -u must be specified (e.g one of 'a', 'r', 't')